lib/components/eureka_list_view.dart

```diff
@@ -65,16 +67,16 @@ class _EurekaListViewState extends State<EurekaListView> {
                   color: Colors.grey,
                 ),
                 _eurekaListViewTextStyle(
-                  value: dateFormat
-                      .format(widget.filteredQuestionsList[index].date)
-                      .toString(),
+                  // timeago is used to format data to return " X days/hours/minutes ago"
+                  value: timeago.format(dateTime),
                   color: Colors.black,
                 ),
               ],
             ),
             _eurekaListViewTextStyle(
               value:
-                  widget.filteredQuestionsList[index].status.toString() == '1'
+                  // status is a boolean
+                  widget.filteredQuestionsList[index].status
                       ? "Active"
                       : "",
               color: Colors.blue,
```

Comment:
Instead of empty string, maybe we should print "Answered" and change font color to green or something like that